### ISSUE TITLE
Limit media embeds in pages to width of media

### ISSUE THREAD

**Issue by allanaaa** [omeka/omeka-s#2037]
I.e. don't let the size of the caption determine the width of the media display, only the width of the media.

This is the quick and dirty solution:

```
.item.resource .caption, .item.resource h3 {
width: 0;
  min-width: 100%;
}
```

That may be subject to themes and their use of flex or grid. I think Omeka should solve this problem out of the box, though, so I'm making this a core issue to start.

**Comment by kimisgold** [comment]
I think having the media width determine the width of the media display can be tricky, especially if you have narrow, portrait media. Our current solution sets a max width on the media display, constraining both media width and caption width.

![image](https://user-images.githubusercontent.com/606764/227617362-17dcf8b1-7940-4f72-a82a-ce8ca8f4dd26.png)

**Comment by allanaaa** [comment]
I don't think that looks good. I'm fine with narrow media having a narrow caption, they really should match each other. Maybe you just want to add more controlled whitespace around them?
